cluster/common.go

```diff
@@ -138,7 +141,9 @@ func (c *CommonClusterBase) getConfig(cluster CommonCluster) ([]byte, error) {
 		log.Debug("k8s config is nil.. load from vault")
 		var loadedConfig []byte
 		configSecret, err := getSecret(cluster.GetOrganizationId(), cluster.GetConfigSecretId())
-		if err != nil {
+		if err == secret.ErrSecretNotExists {
+			return nil, ErrConfigNotExists
```

Comment:
Should we enrich error message with cluster name for which the config was not found or rather do that in all of the callers?